src/flows/components/FlowsIndex.tsx

```diff
@@ -61,32 +107,62 @@ const FlowsIndex = () => {
       titleTag={pageTitleSuffixer([PROJECT_NAME_PLURAL])}
       testID="flows-index"
     >
-      <PageHeader fullWidth={false}>
+      <PageHeader
+        fullWidth={false}
+        className={`${showButtonMode && 'withButtonHeader marginHeader'}`}
+      >
         <Page.Title title={PROJECT_NAME_PLURAL} />
+        {showButtonMode && (
+          <>
+            <PresetFlows buttonMode={showButtonMode} />
+            <Page.ControlBar fullWidth={false}>
+              <Page.ControlBarLeft>
+                <SearchWidget
+                  placeholderText={`Filter ${PROJECT_NAME_PLURAL}...`}
+                  onSearch={setSearch}
+                  searchTerm={search}
+                />
+                <ResourceSortDropdown
+                  resourceType={ResourceType.Flows}
+                  sortDirection={sortOptions.sortDirection}
+                  sortKey={sortOptions.sortKey}
+                  sortType={sortOptions.sortType}
+                  onSelect={setSort}
+                />
+              </Page.ControlBarLeft>
+              <Page.ControlBarRight>
+                {!isFlagEnabled('presetFlows') && <FlowCreateButton />}
+              </Page.ControlBarRight>
+            </Page.ControlBar>
+          </>
+        )}
       </PageHeader>
-      <Page.ControlBar fullWidth={false}>
-        <Page.ControlBarLeft>
-          <SearchWidget
-            placeholderText={`Filter ${PROJECT_NAME_PLURAL}...`}
-            onSearch={setSearch}
-            searchTerm={search}
-          />
-          <ResourceSortDropdown
-            resourceType={ResourceType.Flows}
-            sortDirection={sortOptions.sortDirection}
-            sortKey={sortOptions.sortKey}
-            sortType={sortOptions.sortType}
-            onSelect={setSort}
-          />
-        </Page.ControlBarLeft>
-        <Page.ControlBarRight>
-          {!isFlagEnabled('presetFlows') && <FlowCreateButton />}
-        </Page.ControlBarRight>
-      </Page.ControlBar>
-      <Page.Contents scrollable={true}>
-        {isFlagEnabled('presetFlows') && <PresetFlows />}
-        <FlowCards flows={flowList} search={search} />
-      </Page.Contents>
+      <DapperScrollbars onScroll={scrollHandler} id="scrollFlows">
+        <Page.Contents fullWidth={false} id="fadebox" ref={fadingBoxRef}>
+          <PresetFlows buttonMode={false} />
+          <Page.ControlBar fullWidth={false}>
```

Comment:
is fullWidth explicitly true? if not, remove this one as well

Reply:
yeah it looks like it is unfortunately